crates/serpen/tests/fixtures/ast_rewriting/happy_path/models/user.py

```
@@ -0,0 +1,43 @@
+class UserRole(Enum):
+    """User role enumeration."""
+
+    ADMIN = "admin"
+    USER = "user"
+    GUEST = "guest"
+
+
+class User:
+    """Simple user model."""
+
+    def __init__(self, name: str, email: str, role: UserRole = UserRole.USER):
```

Ruff (0.11.9)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)